app/main.py

```diff
@@ -1,3 +1,7 @@
+from participants.change_participant import apply
+from participants.battle import after_battle
+
+
 KNIGHTS = {
```

Comment:
You could place the knights' config in a separate file to not overload the `app/main.py`.